data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,149 @@
+			label vailheadhome
+			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder if humanity appreciates the luxury such diversity affords them." She looks thoughtful, then gestures toward the ship. "Well, <first>, I promised you a tour of a historic Remnant district. Caelian has perhaps the best example of one, so I would recommend we go there."`
+	on complete
+		conversation
+			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a large platform.`
+			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier." She says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature and will not cause damage, but they can still hurt if you run into them."`
+			`	Moments later you see why the barriers are in place, as a large train whistles into the station at high speed and slows quickly to a stop in front of you. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering or exiting the train. Within moments the doors close and the train starts moving again and you can feel yourself being pressed back into the seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`
+			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: We will only spend a few minutes at top speed before we start decelerating to our destination. I would recommend buckling into the harness as the deceleration will be significant."`
```

Comment:
```suggestion
			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: we will only spend a few minutes at top speed before we start decelerating near our destination. I would recommend buckling into the harness."`
```